### DISCUSSION TITLE
Full landscape word document

### DISCUSSION THREAD

**Discussion by certara-vdamico** [davidgohel/officedown#88] [Q&A] [answered]
Hi! 
I am using an .Rmd file to generate a word document. The document should be entirely landscape format. I have tried using `BLOCK_LANDSCAPE_START` and `BLOCK_LANDSCAPE_STOP` so that the entire document is one big landscape block, but unfortunately the title remains on a portrait page and the last page of the document is portrait, too. Is there any way around this?
I have also tried using a template word document that is a simple, empty landscape file, but that didn't do the trick either (see SO post [here](https://stackoverflow.com/questions/73701051/r-markdown-landscape-only-word-document).
Any help is much appreciated! :)

**Comment by RichardMeyer-Eppler** [comment, marked as answer]
You can set the orientation for the entire document in the YAML options like this:

```
---
output:
  officedown::rdocx_document:
    page_size:
      orient: "landscape"
---
```

**Reply by certara-vdamico** [reply]
That worked perfectly. Thank you!

**Reply by certara-vdamico** [reply]
Hi. As you know, the adjustments to the header helped me perfectly.
The app that generates the report has now been rewritten to an Angular app that uses an R plumber API (instead of an R markdown document with shiny that renders another R markdown document). For some reason, the word document that is rendered, is not landscape. I literally copy-pasted the original .Rmd file into our new repository and am calling in from within the API as I did before from within the shiny .Rmd. Any idea what might be causing the issue?

**Reply by certara-vdamico** [reply]
Actually, seems like a simple RStudio restart resolved it. The document now shows up in landscape again.

**Reply by certara-vdamico** [reply]
Alright so the problem returned and seems to have been a version issue. I can generate a landscape document with {officedown} version 0.2.4 but not 0.3.0.

**Reply by jkylearmstrong** [reply]
okay! I'll go to version 0.2.4

**Comment by PatrickRWright** [comment]
Apparently setting `page_margins` helps in versions `>= 0.3.0`:  
https://stackoverflow.com/a/78941312/2996802
